src/Network/Network/Az.Network.psd1

```diff
@@ -3,751 +3,779 @@
 #
 # Generated by: Microsoft Corporation
 #
-# Generated on: 9/25/2024
+# Generated on: 7/4/2024
```

Comment:
Sanity check the large diff in this file; your only changes should be adding the new cmdlet names. Lines like this date shouldn't change; the NRP release manager should handle that